src/Uri.php

```diff
@@ -624,7 +624,7 @@ private function filterPort($port)
      *
      * @return array
```

Comment:
please also remove phpdocs that become useless due to the type declarations

Reply:
Done.